lib/vm.js

```diff
@@ -355,6 +360,25 @@ function compileFunction(code, params, options = {}) {
   return result.function;
 }
 
+function measureMemory(options = {}, context) {
+  emitExperimentalWarning('vm.measureMemory');
+  validateObject(options, 'options');
+  let mode = options.mode;
+  if (mode === undefined) {
+    mode = constants.measureMemory.mode.SUMMARY;
+  }
+  validateInt32(mode, 'options.mode',
+                constants.measureMemory.mode.SUMMARY,
+                constants.measureMemory.mode.DETAILED);
```

Comment:
I assume they are consecutive numbers so this works but I think explicit check would be better.
Also, this doesn't produce a very clear error - it will say that number in range was expected but nothing about `vm.constants` where to actually get that number.

Reply:
obsolete as we've switched to strings now.